src/sound/Sound.cpp

```diff
@@ -690,7 +696,19 @@ namespace Sound {
 	void Event::Play(const char *fx, float volume_left, float volume_right, Op op)
 	{
 		Stop();
-		eid = PlaySfx(fx, volume_left, volume_right, op);
+		Sample* sample = GetSample(fx);
+		if (sample) {
+			eid = PlaySfxSample(sample, volume_left, volume_right, op);
+		}
+	}
+
+	void Event::PlayMusic(const char *fx, const float volume_left, const float volume_right, Op op)
+	{
+		Stop();
+		Sample* sample = GetSample(fx);
+		if (sample) {
+			eid = PlayMusicSample(sample, volume_left, volume_right, op);
+		}
```

Comment:
This will stop the music even if the new music fails to load.

```suggestion
		Sample* sample = GetSample(fx);
		if (sample) {
			Stop();
			eid = PlayMusicSample(sample, volume_left, volume_right, op);
		}
```

Reply:
I kept the previous behavior of MusicEvent::Play().  I'm thinking this is desirable since having no sample is an error and it will let the user know something is wrong if no music is playing.

Reply:
Yeah, sorry I realized later when i saw another function with similar behavior but forgot to update this comment. 

Is the missing sound file logged? That would be useful for the devs.

Reply:
GetSample() has a commented out SilentWarning() call.

Reply:
Wonder why it's commented out.. anyway, out-of-scope of this PR I guess.